ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/shared/target/TargetContentsGroup.java

```diff
@@ -316,8 +316,9 @@ public void run() {
 				fTree.collapseAll();
 			}
 		});
+		fMenuManager.add(new CopyLocationAction(fTree));
 		Menu contextMenu = fMenuManager.createContextMenu(tree);
-		tree.setMenu(contextMenu);
+		fTree.getTree().setMenu(contextMenu);
```

Comment:
Nice catch.

Reply:
That took much longer to figure out than I care to admit.